includes/class-freemius.php

```diff
@@ -17167,10 +17444,11 @@ private function create_installs_with_user(
          * @author Vova Feldman (@svovaf)
          * @since  1.0.6
          *
-         * @param Freemius      $parent_fs
-         * @param bool|int|null $network_level_or_blog_id True for network level opt-in and integer for opt-in for specified blog in the network.
+         * @param Freemius          $parent_fs
+         * @param bool|int|null     $network_level_or_blog_id True for network level opt-in and integer for opt-in for specified blog in the network.
+         * @param FS_Plugin_License $bundle_license

```

Comment:
@fajardoleo when we introduce new params to already existing methods it's a good practice to add `Since v<version>.` next to the parameter PHP Doc. Also, when the length of the params with the function is long, better to move each param to its own line.